### DISCUSSION TITLE
Flux becomes slower and more Vram intensive with longer prompts?

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [comfyanonymous/ComfyUI#4628] [General]
Every time I use a long prompt it seems my Vram goes out of whack, anyone else notice this too?

**Comment by Melyns** [comment]
I also experience this... i ain't no expert but i think it's due to its architecture, more tokens = less iterations per second.

**Reply by BrechtCorbeel** [reply]
I find it odd, this was never an issue with stable diffusion.

**Comment by comfyanonymous** [comment]
That's perfectly normal. It's the same with all SD versions but the impact is a lot more significant with DiT models like flux because of how they work.

**Reply by BrechtCorbeel** [reply]
I never had any problems with this in SD hence the question because I started trying my SD prompts and then the issues started.

Would it be possible to develop something in comfy to estimate Vram use?, because FLUX + memory usage is getting out of hand and unpredictable.

**Comment by al-swaiti** [comment]
try gguf version of t5 ,, plus  run in  --lowvram mode https://civitai.com/models/668417/t5gguf

**Reply by al-swaiti** [reply]
i run bf16 t5 gguf with flux dev bf16  gguf  on 12 G gpu >>>> but the equation of time go exponential to the power to the ( size of model / size of gpu ) so i used q8 and q6k ,i created interesting merge here try it and plz write your feedback   https://civitai.com/models/682369
